draft-ietf-tls-tls13.md

```diff
@@ -930,14 +933,10 @@ type
 : The higher-level protocol used to process the enclosed fragment.
 
 version
-: The version of the protocol being employed.  This document
-  describes TLS Version 1.3, which uses the version { 3, 4 }.  The
-  version value 3.4 is historical, deriving from the use of {3, 1}
-  for TLS 1.0.  (See {{record-layer-1}}.)  Note that a client that
-  supports multiple versions of TLS may not know what version will
-  be employed before it receives the ServerHello.  See
-  {{backward-compatibility}} for discussion about what record layer
-  version number should be employed for ClientHello.
+: The version of the protocol being employed in the current record.
+  For the initial ClientHello this MUST be { 3, 1 }.
+  For the ServerHello and all other TLS records this MUST be equal to the negotiated version.
+  For TLS 1.3 connections, this version is { 3, 4 }.
```

Comment:
I think we should add the bit about the server being permissive about versions in he client hello from the appendix to here.
Earlier versions of the TLS specification were not fully clear on what the
 record layer version number (TLSPlaintext.version) should contain when sending
 ClientHello (i.e., before it is known which version of the protocol will be
 employed). Thus, TLS servers compliant with this specification MUST accept any value { 03, XX } as the record layer version number for ClientHello.